[bug] compilation error

Hello,

When building on windows, using vcpkg, i get the following error log : 
```
`Change Dir: 'C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel'

Run Build Command(s): "C:/Program Files (x86)/Microsoft Visual Studio/2022/BuildTools/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" -v -v -j17 install
[1/31] C:\Windows\system32\cmd.exe /C "cd /D C:\prog\tools\vcpkg\buildtrees\popsift\x64-windows-release-rel\src\CMakeFiles\popsift.dir\popsift && "C:\Program Files\CMake\bin\cmake.exe" -E make_directory C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/. && "C:\Program Files\CMake\bin\cmake.exe" -D verbose:BOOL=OFF -D build_configuration:STRING=Release -D generated_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_sift_extremum.cu.obj -D generated_cubin_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_sift_extremum.cu.obj.cubin.txt -P C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/popsift_generated_sift_extremum.cu.obj.Release.cmake"
sift_extremum.cu
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
sift_extremum.cu
tmpxft_00004e64_00000000-7_sift_extremum.compute_90.cudafe1.cpp
[2/31] C:\Windows\system32\cmd.exe /C "cd /D C:\prog\tools\vcpkg\buildtrees\popsift\x64-windows-release-rel\src\CMakeFiles\popsift.dir\popsift && "C:\Program Files\CMake\bin\cmake.exe" -E make_directory C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/. && "C:\Program Files\CMake\bin\cmake.exe" -D verbose:BOOL=OFF -D build_configuration:STRING=Release -D generated_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_sift_conf.cu.obj -D generated_cubin_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_sift_conf.cu.obj.cubin.txt -P C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/popsift_generated_sift_conf.cu.obj.Release.cmake"
sift_conf.cu
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
sift_conf.cu
tmpxft_00000e2c_00000000-7_sift_conf.compute_90.cudafe1.cpp
[3/31] C:\Windows\system32\cmd.exe /C "cd /D C:\prog\tools\vcpkg\buildtrees\popsift\x64-windows-release-rel\src\CMakeFiles\popsift.dir\popsift && "C:\Program Files\CMake\bin\cmake.exe" -E make_directory C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/. && "C:\Program Files\CMake\bin\cmake.exe" -D verbose:BOOL=OFF -D build_configuration:STRING=Release -D generated_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_sift_constants.cu.obj -D generated_cubin_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_sift_constants.cu.obj.cubin.txt -P C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/popsift_generated_sift_constants.cu.obj.Release.cmake"
sift_constants.cu
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
sift_constants.cu
tmpxft_00004e7c_00000000-7_sift_constants.compute_90.cudafe1.cpp
[4/31] C:\Windows\system32\cmd.exe /C "cd /D C:\prog\tools\vcpkg\buildtrees\popsift\x64-windows-release-rel\src\CMakeFiles\popsift.dir\popsift && "C:\Program Files\CMake\bin\cmake.exe" -E make_directory C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/. && "C:\Program Files\CMake\bin\cmake.exe" -D verbose:BOOL=OFF -D build_configuration:STRING=Release -D generated_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_s_filtergrid.cu.obj -D generated_cubin_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_s_filtergrid.cu.obj.cubin.txt -P C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/popsift_generated_s_filtergrid.cu.obj.Release.cmake"
FAILED: src/CMakeFiles/popsift.dir/popsift/popsift_generated_s_filtergrid.cu.obj C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/popsift_generated_s_filtergrid.cu.obj 
C:\Windows\system32\cmd.exe /C "cd /D C:\prog\tools\vcpkg\buildtrees\popsift\x64-windows-release-rel\src\CMakeFiles\popsift.dir\popsift && "C:\Program Files\CMake\bin\cmake.exe" -E make_directory C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/. && "C:\Program Files\CMake\bin\cmake.exe" -D verbose:BOOL=OFF -D build_configuration:STRING=Release -D generated_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_s_filtergrid.cu.obj -D generated_cubin_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_s_filtergrid.cu.obj.cubin.txt -P C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/popsift_generated_s_filtergrid.cu.obj.Release.cmake"
s_filtergrid.cu
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3/nvToolsExt.h(941): warning C4002: trop d'arguments pour l'appel de macro de type fonction 'nvtxRangePop'
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail\nvtxImplCore.h(103): warning C4002: trop d'arguments pour l'appel de macro de type fonction 'nvtxRangePop'
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3/nvToolsExt.h(941): warning C4002: trop d'arguments pour l'appel de macro de type fonction 'nvtxRangePop'
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail\nvtxImplCore.h(103): warning C4002: trop d'arguments pour l'appel de macro de type fonction 'nvtxRangePop'
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3/nvToolsExt.h(941): warning C4002: trop d'arguments pour l'appel de macro de type fonction 'nvtxRangePop'
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail\nvtxImplCore.h(103): warning C4002: trop d'arguments pour l'appel de macro de type fonction 'nvtxRangePop'
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3/nvToolsExt.h(941): warning C4002: trop d'arguments pour l'appel de macro de type fonction 'nvtxRangePop'
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail\nvtxImplCore.h(103): warning C4002: trop d'arguments pour l'appel de macro de type fonction 'nvtxRangePop'
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3/nvToolsExt.h(941): warning C4002: trop d'arguments pour l'appel de macro de type fonction 'nvtxRangePop'
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail\nvtxImplCore.h(103): warning C4002: trop d'arguments pour l'appel de macro de type fonction 'nvtxRangePop'
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3/nvToolsExt.h(941): warning C4002: trop d'arguments pour l'appel de macro de type fonction 'nvtxRangePop'
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail\nvtxImplCore.h(103): warning C4002: trop d'arguments pour l'appel de macro de type fonction 'nvtxRangePop'
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3/nvToolsExt.h(941): warning C4002: trop d'arguments pour l'appel de macro de type fonction 'nvtxRangePop'
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail\nvtxImplCore.h(103): warning C4002: trop d'arguments pour l'appel de macro de type fonction 'nvtxRangePop'
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3/nvToolsExt.h(941): warning C4002: trop d'arguments pour l'appel de macro de type fonction 'nvtxRangePop'
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail\nvtxImplCore.h(103): warning C4002: trop d'arguments pour l'appel de macro de type fonction 'nvtxRangePop'
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3/nvToolsExt.h(941): warning C4002: trop d'arguments pour l'appel de macro de type fonction 'nvtxRangePop'
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail\nvtxImplCore.h(103): warning C4002: trop d'arguments pour l'appel de macro de type fonction 'nvtxRangePop'
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3/nvToolsExt.h(941): warning C4002: trop d'arguments pour l'appel de macro de type fonction 'nvtxRangePop'
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail\nvtxImplCore.h(103): warning C4002: trop d'arguments pour l'appel de macro de type fonction 'nvtxRangePop'
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3/nvToolsExt.h(941): warning C4002: trop d'arguments pour l'appel de macro de type fonction 'nvtxRangePop'
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail\nvtxImplCore.h(103): warning C4002: trop d'arguments pour l'appel de macro de type fonction 'nvtxRangePop'
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3/nvToolsExt.h(941): warning C4002: trop d'arguments pour l'appel de macro de type fonction 'nvtxRangePop'
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail\nvtxImplCore.h(103): warning C4002: trop d'arguments pour l'appel de macro de type fonction 'nvtxRangePop'
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3/nvToolsExt.h(890): error: expected an identifier
  __inline static int __stdcall ;
                                ^

Error: inline specifier allowed on function declarations only

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3/nvToolsExt.h(890): error: the modifier "__inline" is not allowed on this declaration
  __inline static int __stdcall ;
                                ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3/nvToolsExt.h(941): error: expected an identifier
  __inline static int __stdcall ;
                                ^

Error: inline specifier allowed on function declarations only

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3/nvToolsExt.h(941): error: the modifier "__inline" is not allowed on this declaration
  __inline static int __stdcall ;
                                ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail\nvtxImplCore.h(82): error: expected an identifier
  {
  ^

Error: inline specifier allowed on function declarations only

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail\nvtxImplCore.h(82): error: the modifier "__inline" is not allowed on this declaration
  {
  ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail\nvtxImplCore.h(84): error: type name is not allowed
      nvtxRangePushA_impl_fntype local = nvtxGlobals_v3.nvtxRangePushA_impl_fnptr;
      ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail\nvtxImplCore.h(84): error: expected a ","
      nvtxRangePushA_impl_fntype local = nvtxGlobals_v3.nvtxRangePushA_impl_fnptr;
                                 ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail\nvtxImplCore.h(84): error: expected a "}"
      nvtxRangePushA_impl_fntype local = nvtxGlobals_v3.nvtxRangePushA_impl_fnptr;
                                                                                 ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail\nvtxImplCore.h(85): error: expected a declaration
      if(local!=0)
      ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail\nvtxImplCore.h(87): error: expected a declaration
      else
      ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail\nvtxImplCore.h(104): error: expected an identifier
  {
  ^

Error: inline specifier allowed on function declarations only

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail\nvtxImplCore.h(104): error: the modifier "__inline" is not allowed on this declaration
  {
  ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail\nvtxImplCore.h(106): error: type name is not allowed
      nvtxRangePop_impl_fntype local = nvtxGlobals_v3.nvtxRangePop_impl_fnptr;
      ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail\nvtxImplCore.h(106): error: expected a ","
      nvtxRangePop_impl_fntype local = nvtxGlobals_v3.nvtxRangePop_impl_fnptr;
                               ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail\nvtxImplCore.h(106): error: expected a "}"
      nvtxRangePop_impl_fntype local = nvtxGlobals_v3.nvtxRangePop_impl_fnptr;
                                                                             ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail\nvtxImplCore.h(107): error: expected a declaration
      if(local!=0)
      ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail\nvtxImplCore.h(109): error: expected a declaration
      else
      ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail\nvtxImplCore.h(112): error: expected a declaration
  }
  ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail/nvtxImpl.h(418): warning #12-D: parsing restarts here after previous syntax error
          break;
               ^

Remark: The warnings can be suppressed with "-diag-suppress <warning-number>"

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail/nvtxImpl.h(419): error: expected a declaration
      case NVTX_CB_MODULE_CORE2:
      ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail/nvtxImpl.h(421): error: this declaration has no storage class or type specifier
          bytes = (unsigned int)sizeof(nvtxGlobals_v3.functionTable_CORE2);
          ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail/nvtxImpl.h(422): error: expected a declaration
          break;
          ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail/nvtxImpl.h(423): error: expected a declaration
      case NVTX_CB_MODULE_SYNC:
      ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail/nvtxImpl.h(425): error: this declaration has no storage class or type specifier
          bytes = (unsigned int)sizeof(nvtxGlobals_v3.functionTable_SYNC);
          ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail/nvtxImpl.h(425): error: variable "bytes" has already been defined (previous definition at line 421)
          bytes = (unsigned int)sizeof(nvtxGlobals_v3.functionTable_SYNC);
          ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail/nvtxImpl.h(426): error: expected a declaration
          break;
          ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail/nvtxImpl.h(427): error: this declaration has no storage class or type specifier
      default: return 0;
      ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail/nvtxImpl.h(427): error: expected a ";"
      default: return 0;
             ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail/nvtxImpl.h(428): error: expected a declaration
      }
      ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail/nvtxImpl.h(431): warning #12-D: parsing restarts here after previous syntax error
          *out_size = (bytes / (unsigned int)sizeof(NvtxFunctionPointer*)) - 1;
                                                                              ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail/nvtxImpl.h(433): error: expected a declaration
      if (out_table)
      ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail/nvtxImpl.h(436): error: expected a declaration
      return 1;
      ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail/nvtxImpl.h(437): error: expected a declaration
  }
  ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail\nvtxInitDefs.h(429): warning #12-D: parsing restarts here after previous syntax error
  extern "C" inline void nvtxSetInitFunctionsToNoops_v3(int forceAllToNoops);
                                                                            ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\nvtx3\nvtxDetail/nvtxImpl.h(468): error: expected a declaration
  } 
  ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(661): warning #12-D: parsing restarts here after previous syntax error
    {};
      ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(682): error: expected a declaration
    } 
    ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(934): warning #12-D: parsing restarts here after previous syntax error
      ~domain() = default;
                         ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(936): error: expected a declaration
    private:
    ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(938): error: expected a declaration
    };
    ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(954): error: explicit type is missing ("int" assumed)
    inline domain const& domain::get<domain::global>() noexcept
    ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(954): error: "domain" is not a class or function template name in the current scope
    inline domain const& domain::get<domain::global>() noexcept
           ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(954): error: expected a ";"
    inline domain const& domain::get<domain::global>() noexcept
                  ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(987): warning #12-D: parsing restarts here after previous syntax error
      component_type green{}; 
                            ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(988): error: identifier "component_type" is undefined
      component_type blue{}; 
      ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(989): error: expected a declaration
    };
    ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(996): error: not a class or struct name
    struct argb final : rgb
                        ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1010): error: identifier "component_type" is undefined
      constexpr argb(component_type alpha_, component_type red_, component_type green_, component_type blue_) noexcept
                     ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1010): error: identifier "component_type" is undefined
      constexpr argb(component_type alpha_, component_type red_, component_type green_, component_type blue_) noexcept
                                            ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1010): error: identifier "component_type" is undefined
      constexpr argb(component_type alpha_, component_type red_, component_type green_, component_type blue_) noexcept
                                                                 ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1010): error: identifier "component_type" is undefined
      constexpr argb(component_type alpha_, component_type red_, component_type green_, component_type blue_) noexcept
                                                                                        ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1015): error: identifier "component_type" is undefined
      component_type alpha{}; 
      ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1011): error: "rgb" is not a nonstatic data member or base class of class "argb"
          : rgb{red_, green_, blue_}
            ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1072): error: identifier "rgb" is undefined
      constexpr color(rgb rgb_) noexcept
                      ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1061): error: class "argb" has no member "red"
          : color{from_bytes_msb_to_lsb(argb_.alpha, argb_.red, argb_.green, argb_.blue)}
                                                           ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1061): error: class "argb" has no member "green"
          : color{from_bytes_msb_to_lsb(argb_.alpha, argb_.red, argb_.green, argb_.blue)}
                                                                      ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1061): error: class "argb" has no member "blue"
          : color{from_bytes_msb_to_lsb(argb_.alpha, argb_.red, argb_.green, argb_.blue)}
                                                                                   ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1226): error: name followed by "::" must be a class or namespace name
    template <typename D = domain::global>
                           ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1267): error: name followed by "::" must be a class or namespace name
        typename std::enable_if<detail::is_c_string<decltype(C::name)>::value && detail::is_uint32<decltype(C::id)>::value,
                                ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1267): error: expected an identifier
        typename std::enable_if<detail::is_c_string<decltype(C::name)>::value && detail::is_uint32<decltype(C::id)>::value,
                                                    ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1267): error: expected a ">"
        typename std::enable_if<detail::is_c_string<decltype(C::name)>::value && detail::is_uint32<decltype(C::id)>::value,
                                                    ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1267): error: name followed by "::" must be a class or namespace name
        typename std::enable_if<detail::is_c_string<decltype(C::name)>::value && detail::is_uint32<decltype(C::id)>::value,
                                                                                 ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1267): error: expected an identifier
        typename std::enable_if<detail::is_c_string<decltype(C::name)>::value && detail::is_uint32<decltype(C::id)>::value,
                                                                                                   ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1267): error: a nontype template parameter cannot have rvalue reference type
        typename std::enable_if<detail::is_c_string<decltype(C::name)>::value && detail::is_uint32<decltype(C::id)>::value,
        ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1267): error: expected a "," or ">"
        typename std::enable_if<detail::is_c_string<decltype(C::name)>::value && detail::is_uint32<decltype(C::id)>::value,
                                                                                                   ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1267): error: the global scope has no "value"
        typename std::enable_if<detail::is_c_string<decltype(C::name)>::value && detail::is_uint32<decltype(C::id)>::value,
                                                                                                                     ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1267): error: this declaration has no storage class or type specifier
        typename std::enable_if<detail::is_c_string<decltype(C::name)>::value && detail::is_uint32<decltype(C::id)>::value,
                                                                                                                   ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1267): error: member "named_category_in<D>::value" is not a valid class member template
        typename std::enable_if<detail::is_c_string<decltype(C::name)>::value && detail::is_uint32<decltype(C::id)>::value,
                                                                                                                   ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1267): error: expected a ";"
        typename std::enable_if<detail::is_c_string<decltype(C::name)>::value && detail::is_uint32<decltype(C::id)>::value,
                                                                                                                          ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1350): warning #12-D: parsing restarts here after previous syntax error
      };
       ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1366): error: name followed by "::" must be a class or namespace name
        nvtxDomainNameCategoryW(domain::get<D>(), get_id(), name);
                                ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1366): error: expected an expression
        nvtxDomainNameCategoryW(domain::get<D>(), get_id(), name);
                                               ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1378): error: name followed by "::" must be a class or namespace name
    using named_category = named_category_in<domain::global>;
                                             ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1426): error: name followed by "::" must be a class or namespace name
    template <typename D = domain::global>
                           ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1464): error: name followed by "::" must be a class or namespace name
      template <typename M, typename std::enable_if<detail::is_c_string<decltype(M::message)>::value, int>::type = 0>
                                                    ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1464): error: expected an identifier
      template <typename M, typename std::enable_if<detail::is_c_string<decltype(M::message)>::value, int>::type = 0>
                                                                        ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1464): error: expected a ">"
      template <typename M, typename std::enable_if<detail::is_c_string<decltype(M::message)>::value, int>::type = 0>
                                                                        ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1464): error: the global scope has no "type"
      template <typename M, typename std::enable_if<detail::is_c_string<decltype(M::message)>::value, int>::type = 0>
                                                                                                            ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1464): error: this declaration has no storage class or type specifier
      template <typename M, typename std::enable_if<detail::is_c_string<decltype(M::message)>::value, int>::type = 0>
                                                                                                          ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1464): error: member "registered_string_in<D>::type" is not a valid class member template
      template <typename M, typename std::enable_if<detail::is_c_string<decltype(M::message)>::value, int>::type = 0>
                                                                                                          ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1464): error: data member initializer is not allowed
      template <typename M, typename std::enable_if<detail::is_c_string<decltype(M::message)>::value, int>::type = 0>
                                                                                                                 ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1581): warning #12-D: parsing restarts here after previous syntax error
      registered_string_in() noexcept {};
                                        ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(1599): error: name followed by "::" must be a class or namespace name
    using registered_string = registered_string_in<domain::global>;
                                                   ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(2099): error: name followed by "::" must be a class or namespace name
    template <class D = domain::global>
                        ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(2120): error: name followed by "::" must be a class or namespace name
        nvtxDomainRangePushEx(domain::get<D>(), attr.get());
                              ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(2120): error: expected an expression
        nvtxDomainRangePushEx(domain::get<D>(), attr.get());
                                             ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(2179): error: name followed by "::" must be a class or namespace name
        nvtxDomainRangePop(domain::get<D>());
                           ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(2179): error: expected an expression
        nvtxDomainRangePop(domain::get<D>());
                                          ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(2188): error: name followed by "::" must be a class or namespace name
    using scoped_range = scoped_range_in<domain::global>;
                                         ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(2194): error: name followed by "::" must be a class or namespace name
    template <typename D = domain::global>
                           ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(2212): error: name followed by "::" must be a class or namespace name
        nvtxDomainRangePushEx(domain::get<D>(), attr.get());
                              ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(2212): error: expected an expression
        nvtxDomainRangePushEx(domain::get<D>(), attr.get());
                                             ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(2222): error: name followed by "::" must be a class or namespace name
          nvtxDomainRangePop(domain::get<D>());
                             ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(2222): error: expected an expression
          nvtxDomainRangePop(domain::get<D>());
                                            ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(2362): error: name followed by "::" must be a class or namespace name
    template <typename D = domain::global>
                           ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(2366): error: name followed by "::" must be a class or namespace name
      return range_handle{nvtxDomainRangeStartEx(domain::get<D>(), attr.get())};
                                                 ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(2366): error: expected an expression
      return range_handle{nvtxDomainRangeStartEx(domain::get<D>(), attr.get())};
                                                                ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(2403): error: name followed by "::" must be a class or namespace name
    template <typename D = domain::global, typename... Args>
                           ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(2442): error: name followed by "::" must be a class or namespace name
      return start_range_in<domain::global>(attr);
                            ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(2480): error: name followed by "::" must be a class or namespace name
      return start_range_in<domain::global>(args...);
                            ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(2501): error: name followed by "::" must be a class or namespace name
    template <typename D = domain::global>
                           ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(2505): error: name followed by "::" must be a class or namespace name
      nvtxDomainRangeEnd(domain::get<D>(), r.get_value());
                         ^

C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.8\include\cub\detail\nvtx3.hpp(2505): error: expected an expression
      nvtxDomainRangeEnd(domain::get<D>(), r.get_value());
                                        ^

Error limit reached.
100 errors detected in the compilation of "C:/prog/tools/vcpkg/buildtrees/popsift/src/v0.9-ba1d7e1254.clean/src/popsift/s_filtergrid.cu".
Compilation terminated.
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
s_filtergrid.cu
CMake Error at popsift_generated_s_filtergrid.cu.obj.Release.cmake:278 (message):
  Error generating file
  C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_s_filtergrid.cu.obj


[5/31] C:\Windows\system32\cmd.exe /C "cd /D C:\prog\tools\vcpkg\buildtrees\popsift\x64-windows-release-rel\src\CMakeFiles\popsift.dir\popsift && "C:\Program Files\CMake\bin\cmake.exe" -E make_directory C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/. && "C:\Program Files\CMake\bin\cmake.exe" -D verbose:BOOL=OFF -D build_configuration:STRING=Release -D generated_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_sift_octave.cu.obj -D generated_cubin_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_sift_octave.cu.obj.cubin.txt -P C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/popsift_generated_sift_octave.cu.obj.Release.cmake"
sift_octave.cu
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
sift_octave.cu
tmpxft_00004f40_00000000-7_sift_octave.compute_90.cudafe1.cpp
[6/31] C:\Windows\system32\cmd.exe /C "cd /D C:\prog\tools\vcpkg\buildtrees\popsift\x64-windows-release-rel\src\CMakeFiles\popsift.dir\popsift && "C:\Program Files\CMake\bin\cmake.exe" -E make_directory C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/. && "C:\Program Files\CMake\bin\cmake.exe" -D verbose:BOOL=OFF -D build_configuration:STRING=Release -D generated_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_gauss_filter.cu.obj -D generated_cubin_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_gauss_filter.cu.obj.cubin.txt -P C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/popsift_generated_gauss_filter.cu.obj.Release.cmake"
gauss_filter.cu
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
gauss_filter.cu
tmpxft_000016f4_00000000-7_gauss_filter.compute_90.cudafe1.cpp
[7/31] C:\Windows\system32\cmd.exe /C "cd /D C:\prog\tools\vcpkg\buildtrees\popsift\x64-windows-release-rel\src\CMakeFiles\popsift.dir\popsift && "C:\Program Files\CMake\bin\cmake.exe" -E make_directory C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/. && "C:\Program Files\CMake\bin\cmake.exe" -D verbose:BOOL=OFF -D build_configuration:STRING=Release -D generated_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_s_image.cu.obj -D generated_cubin_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_s_image.cu.obj.cubin.txt -P C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/popsift_generated_s_image.cu.obj.Release.cmake"
s_image.cu
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
s_image.cu
tmpxft_00004ef8_00000000-7_s_image.compute_90.cudafe1.cpp
[8/31] C:\Windows\system32\cmd.exe /C "cd /D C:\prog\tools\vcpkg\buildtrees\popsift\x64-windows-release-rel\src\CMakeFiles\popsift.dir\popsift && "C:\Program Files\CMake\bin\cmake.exe" -E make_directory C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/. && "C:\Program Files\CMake\bin\cmake.exe" -D verbose:BOOL=OFF -D build_configuration:STRING=Release -D generated_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_s_pyramid_build_ra.cu.obj -D generated_cubin_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_s_pyramid_build_ra.cu.obj.cubin.txt -P C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/popsift_generated_s_pyramid_build_ra.cu.obj.Release.cmake"
s_pyramid_build_ra.cu
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
s_pyramid_build_ra.cu
tmpxft_000030fc_00000000-7_s_pyramid_build_ra.compute_90.cudafe1.cpp
[9/31] C:\Windows\system32\cmd.exe /C "cd /D C:\prog\tools\vcpkg\buildtrees\popsift\x64-windows-release-rel\src\CMakeFiles\popsift.dir\popsift && "C:\Program Files\CMake\bin\cmake.exe" -E make_directory C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/. && "C:\Program Files\CMake\bin\cmake.exe" -D verbose:BOOL=OFF -D build_configuration:STRING=Release -D generated_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_s_pyramid_build_ai.cu.obj -D generated_cubin_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_s_pyramid_build_ai.cu.obj.cubin.txt -P C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/popsift_generated_s_pyramid_build_ai.cu.obj.Release.cmake"
s_pyramid_build_ai.cu
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
s_pyramid_build_ai.cu
tmpxft_000043ac_00000000-7_s_pyramid_build_ai.compute_90.cudafe1.cpp
[10/31] C:\Windows\system32\cmd.exe /C "cd /D C:\prog\tools\vcpkg\buildtrees\popsift\x64-windows-release-rel\src\CMakeFiles\popsift.dir\popsift && "C:\Program Files\CMake\bin\cmake.exe" -E make_directory C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/. && "C:\Program Files\CMake\bin\cmake.exe" -D verbose:BOOL=OFF -D build_configuration:STRING=Release -D generated_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_s_pyramid_build_aa.cu.obj -D generated_cubin_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_s_pyramid_build_aa.cu.obj.cubin.txt -P C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/popsift_generated_s_pyramid_build_aa.cu.obj.Release.cmake"
s_pyramid_build_aa.cu
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
s_pyramid_build_aa.cu
tmpxft_000016cc_00000000-7_s_pyramid_build_aa.compute_90.cudafe1.cpp
[11/31] C:\Windows\system32\cmd.exe /C "cd /D C:\prog\tools\vcpkg\buildtrees\popsift\x64-windows-release-rel\src\CMakeFiles\popsift.dir\popsift && "C:\Program Files\CMake\bin\cmake.exe" -E make_directory C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/. && "C:\Program Files\CMake\bin\cmake.exe" -D verbose:BOOL=OFF -D build_configuration:STRING=Release -D generated_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_s_pyramid_build.cu.obj -D generated_cubin_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_s_pyramid_build.cu.obj.cubin.txt -P C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/popsift_generated_s_pyramid_build.cu.obj.Release.cmake"
s_pyramid_build.cu
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
s_pyramid_build.cu
tmpxft_00002898_00000000-7_s_pyramid_build.compute_90.cudafe1.cpp
[12/31] C:\Windows\system32\cmd.exe /C "cd /D C:\prog\tools\vcpkg\buildtrees\popsift\x64-windows-release-rel\src\CMakeFiles\popsift.dir\popsift && "C:\Program Files\CMake\bin\cmake.exe" -E make_directory C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/. && "C:\Program Files\CMake\bin\cmake.exe" -D verbose:BOOL=OFF -D build_configuration:STRING=Release -D generated_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_features.cu.obj -D generated_cubin_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_features.cu.obj.cubin.txt -P C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/popsift_generated_features.cu.obj.Release.cmake"
features.cu
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
features.cu
tmpxft_00004bc8_00000000-7_features.compute_90.cudafe1.cpp
[13/31] C:\Windows\system32\cmd.exe /C "cd /D C:\prog\tools\vcpkg\buildtrees\popsift\x64-windows-release-rel\src\CMakeFiles\popsift.dir\popsift && "C:\Program Files\CMake\bin\cmake.exe" -E make_directory C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/. && "C:\Program Files\CMake\bin\cmake.exe" -D verbose:BOOL=OFF -D build_configuration:STRING=Release -D generated_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_sift_desc.cu.obj -D generated_cubin_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_sift_desc.cu.obj.cubin.txt -P C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/popsift_generated_sift_desc.cu.obj.Release.cmake"
sift_desc.cu
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
sift_desc.cu
tmpxft_00000ab8_00000000-7_sift_desc.compute_90.cudafe1.cpp
[14/31] C:\Windows\system32\cmd.exe /C "cd /D C:\prog\tools\vcpkg\buildtrees\popsift\x64-windows-release-rel\src\CMakeFiles\popsift.dir\popsift && "C:\Program Files\CMake\bin\cmake.exe" -E make_directory C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/. && "C:\Program Files\CMake\bin\cmake.exe" -D verbose:BOOL=OFF -D build_configuration:STRING=Release -D generated_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_s_pyramid_fixed.cu.obj -D generated_cubin_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_s_pyramid_fixed.cu.obj.cubin.txt -P C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/popsift_generated_s_pyramid_fixed.cu.obj.Release.cmake"
s_pyramid_fixed.cu
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
s_pyramid_fixed.cu
tmpxft_00001884_00000000-7_s_pyramid_fixed.compute_90.cudafe1.cpp
[15/31] C:\Windows\system32\cmd.exe /C "cd /D C:\prog\tools\vcpkg\buildtrees\popsift\x64-windows-release-rel\src\CMakeFiles\popsift.dir\popsift && "C:\Program Files\CMake\bin\cmake.exe" -E make_directory C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/. && "C:\Program Files\CMake\bin\cmake.exe" -D verbose:BOOL=OFF -D build_configuration:STRING=Release -D generated_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_sift_pyramid.cu.obj -D generated_cubin_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_sift_pyramid.cu.obj.cubin.txt -P C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/popsift_generated_sift_pyramid.cu.obj.Release.cmake"
sift_pyramid.cu
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
C:\prog\tools\vcpkg\buildtrees\popsift\src\v0.9-ba1d7e1254.clean\src\popsift\sift_pyramid.cu(419): warning #20091-D: a __device__ variable "M_PI2" cannot be directly read in a host function
              dom_ori = dom_ori / M_PI2 * 360;
                                  ^

Remark: The warnings can be suppressed with "-diag-suppress <warning-number>"

nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
sift_pyramid.cu
tmpxft_000046bc_00000000-7_sift_pyramid.compute_90.cudafe1.cpp
[16/31] C:\Windows\system32\cmd.exe /C "cd /D C:\prog\tools\vcpkg\buildtrees\popsift\x64-windows-release-rel\src\CMakeFiles\popsift.dir\popsift && "C:\Program Files\CMake\bin\cmake.exe" -E make_directory C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/. && "C:\Program Files\CMake\bin\cmake.exe" -D verbose:BOOL=OFF -D build_configuration:STRING=Release -D generated_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_s_orientation.cu.obj -D generated_cubin_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_s_orientation.cu.obj.cubin.txt -P C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/popsift_generated_s_orientation.cu.obj.Release.cmake"
s_orientation.cu
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
s_orientation.cu
tmpxft_00003e2c_00000000-7_s_orientation.compute_90.cudafe1.cpp
[17/31] C:\Windows\system32\cmd.exe /C "cd /D C:\prog\tools\vcpkg\buildtrees\popsift\x64-windows-release-rel\src\CMakeFiles\popsift.dir\popsift && "C:\Program Files\CMake\bin\cmake.exe" -E make_directory C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/. && "C:\Program Files\CMake\bin\cmake.exe" -D verbose:BOOL=OFF -D build_configuration:STRING=Release -D generated_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_s_extrema.cu.obj -D generated_cubin_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_s_extrema.cu.obj.cubin.txt -P C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/popsift_generated_s_extrema.cu.obj.Release.cmake"
s_extrema.cu
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
s_extrema.cu
tmpxft_00004198_00000000-7_s_extrema.compute_90.cudafe1.cpp
[18/31] C:\Windows\system32\cmd.exe /C "cd /D C:\prog\tools\vcpkg\buildtrees\popsift\x64-windows-release-rel\src\CMakeFiles\popsift.dir\popsift && "C:\Program Files\CMake\bin\cmake.exe" -E make_directory C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/. && "C:\Program Files\CMake\bin\cmake.exe" -D verbose:BOOL=OFF -D build_configuration:STRING=Release -D generated_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_s_desc_loop.cu.obj -D generated_cubin_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_s_desc_loop.cu.obj.cubin.txt -P C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/popsift_generated_s_desc_loop.cu.obj.Release.cmake"
s_desc_loop.cu
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
s_desc_loop.cu
tmpxft_00001df0_00000000-7_s_desc_loop.compute_90.cudafe1.cpp
[19/31] C:\Windows\system32\cmd.exe /C "cd /D C:\prog\tools\vcpkg\buildtrees\popsift\x64-windows-release-rel\src\CMakeFiles\popsift.dir\popsift && "C:\Program Files\CMake\bin\cmake.exe" -E make_directory C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/. && "C:\Program Files\CMake\bin\cmake.exe" -D verbose:BOOL=OFF -D build_configuration:STRING=Release -D generated_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_s_desc_iloop.cu.obj -D generated_cubin_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_s_desc_iloop.cu.obj.cubin.txt -P C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/popsift_generated_s_desc_iloop.cu.obj.Release.cmake"
s_desc_iloop.cu
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
s_desc_iloop.cu
tmpxft_000038a8_00000000-7_s_desc_iloop.compute_90.cudafe1.cpp
[20/31] C:\Windows\system32\cmd.exe /C "cd /D C:\prog\tools\vcpkg\buildtrees\popsift\x64-windows-release-rel\src\CMakeFiles\popsift.dir\popsift && "C:\Program Files\CMake\bin\cmake.exe" -E make_directory C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/. && "C:\Program Files\CMake\bin\cmake.exe" -D verbose:BOOL=OFF -D build_configuration:STRING=Release -D generated_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_s_desc_grid.cu.obj -D generated_cubin_file:STRING=C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/./popsift_generated_s_desc_grid.cu.obj.cubin.txt -P C:/prog/tools/vcpkg/buildtrees/popsift/x64-windows-release-rel/src/CMakeFiles/popsift.dir/popsift/popsift_generated_s_desc_grid.cu.obj.Release.cmake"
s_desc_grid.cu
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
nvcc warning : Support for offline compilation for architectures prior to '<compute/sm/lto>_75' will be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
s_desc_grid.cu
tmpxft_00003cb4_00000000-7_s_desc_grid.compute_90.cudafe1.cpp
ninja: build stopped: subcommand failed.
`
```
My vcpkg is up to date, using visual studio 2022 and cuda toolkit 12.8.

Any guess ?

Thanks

--- a/.markdownlint.json
+++ b/.markdownlint.json
@@ -1,0 +1,7 @@
+{
+  "default": true,
+  "MD013": false,
+  "MD024": false,
+  "MD033": false,
+  "MD041": false
+}

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -15,50 +15,76 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Removed
 
-## [1.0.0] - YYYY-MM-DD
-
-## 2025
-
-- Added vlFeat's descriptor extraction method as an option [PR](https://github.com/alicevision/popsift/pull/167)
-
-## 2024
-
-- CMake: CUDA as first-order language, different CC selection
+## [0.10]
 
 ### Added
-- Improved checks for CUDA textures [PR](https://github.com/alicevision/popsift/pull/89)
-- CMake: Improved support for all Cuda CC [PR](https://github.com/alicevision/popsift/pull/75)
-- CMake: support for cuda 11 [PR](https://github.com/alicevision/popsift/pull/103)
-- Support for Cuda CC 7 cards (RTX 2080) [PR](https://github.com/alicevision/popsift/pull/67)
-- Support for Boost 1.70 [PR](https://github.com/alicevision/popsift/pull/65)
-- Support for device selection and multiple GPUs [PR](https://github.com/alicevision/popsift/pull/121)
 
-### Fixed
-- CMake: fixes to allow building on Windows using vcpkg [PR](https://github.com/alicevision/popsift/pull/92)
-- Fix race condition [PR](https://github.com/alicevision/popsift/pull/82)
+- Support for device selection and multiple GPUs [PR](https://github.com/alicevision/popsift/pull/121)
+- Support for CUDA versions through 12.1 [PR](https://github.com/alicevision/popsift/pull/146)
+- GitHub Actions for CI [PR](https://github.com/alicevision/popsift/pull/149)
+- Missing thrust include [PR](https://github.com/alicevision/popsift/pull/135)
+- Required thrust include for `s_filtergrid.cu` [PR](https://github.com/alicevision/popsift/pull/144)
+- Added vlFeat's descriptor extraction method as an option [PR](https://github.com/alicevision/popsift/pull/167)
 
 ### Changed
-- Improved resource releasing [PR](https://github.com/alicevision/popsift/pull/71)
+
+- CMake: CUDA as first-order language, different CC selection [PR](https://github.com/alicevision/popsift/pull/156)
+- CMake: SM 86 only for CUDA >= 11.1 [PR](https://github.com/alicevision/popsift/pull/116)
+- CMake: Suppress deprecated CUDA SM warnings [PR](https://github.com/alicevision/popsift/pull/124)
+- Replace exit() calls with thrown exceptions [PR](https://github.com/alicevision/popsift/pull/141)
+
+### Fixed
+
+- Errors with large images [PR](https://github.com/alicevision/popsift/pull/145)
+- CMake: Explicit CC list for Jetson and Tegra platforms [PR](https://github.com/alicevision/popsift/pull/163)
+- Remove broken and unused code path from L2 normalization [PR](https://github.com/alicevision/popsift/pull/166)
+- Do not use NVTX [PR](https://github.com/alicevision/popsift/pull/162)
+- Documentation: Biblio build [PR](https://github.com/alicevision/popsift/pull/132)
+
+### Documentation
+
+- Update with vcpkg package [PR](https://github.com/alicevision/popsift/pull/129)
+
+## [0.9] - 2021-02-13
+
+### Added
+
+- Support for CUDA 11 [PR](https://github.com/alicevision/popsift/pull/103)
+- Improved checks for CUDA textures [PR](https://github.com/alicevision/popsift/pull/89)
+- Documentation: ReadTheDocs integration [PR](https://github.com/alicevision/popsift/pull/100)
+- CMake: Support for SM86 compute capability [PR](https://github.com/alicevision/popsift/pull/113)
+
+### Changed
+
+- Docker: Updated CMake and split Docker in 2 parts [PR](https://github.com/alicevision/popsift/pull/87)
+- Code cleanup and improvements [PR](https://github.com/alicevision/popsift/pull/95)
+
+### Fixed
+
+- Race conditions identified by racecheck [PR](https://github.com/alicevision/popsift/pull/82)
+- CMake: Windows build fixes [PR](https://github.com/alicevision/popsift/pull/92)
+- Memory management: pair `malloc` with `free`, not `delete` [PR](https://github.com/alicevision/popsift/pull/108)
+- CUDA: Pass correct shared memory size to orientation kernel [PR](https://github.com/alicevision/popsift/pull/109)
+- Testing paths fixes [PR](https://github.com/alicevision/popsift/pull/104)
+- Remove stale undefined function [PR](https://github.com/alicevision/popsift/pull/90)
 
 ### Removed
-- Remove boost dependency from the main library [PR](https://github.com/alicevision/popsift/pull/81)
 
+- Remove boost dependency from core popsift library [PR](https://github.com/alicevision/popsift/pull/81)
 
 ## 2019
 
 - Bugfix: Support for images with different resolutions [PR](https://github.com/alicevision/popsift/pull/58)
 
-
 ## 2018
 
 - CMake: Auto-build export symbols for shared libs on Windows [PR](https://github.com/alicevision/popsift/pull/54)
 - Bugfix: freeing page-aligned memory on Win32 [PR](https://github.com/alicevision/popsift/pull/53)
-- Paper published @MMSys18 (https://dl.acm.org/doi/10.1145/3204949.3208136)
+- Paper published @MMSys18 (<https://dl.acm.org/doi/10.1145/3204949.3208136>)
 - Docker support [PR](https://github.com/alicevision/popsift/pull/46)
 - Appveyor CI windows [PR](https://github.com/alicevision/popsift/pull/41)
 - Support for Cuda 9 [PR](https://github.com/alicevision/popsift/pull/38)
 - Thrust with Cuda 7 [PR](https://github.com/alicevision/popsift/pull/35)
-
 
 ## 2017
 
@@ -67,9 +93,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Support asynchronous SIFT extraction [PR](https://github.com/alicevision/popsift/pull/22)
 - Windows port [PR](https://github.com/alicevision/popsift/pull/18)
 
-
 ## 2016
 
 - Switch to modern CMake [PR](https://github.com/alicevision/popsift/pull/14)
 - Travis CI Linux [PR](https://github.com/alicevision/popsift/pull/8)
- - First open-source release
+- First open-source release

--- a/CHANGES.md
+++ b/CHANGES.md
@@ -25,6 +25,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Missing thrust include [PR](https://github.com/alicevision/popsift/pull/135)
 - Required thrust include for `s_filtergrid.cu` [PR](https://github.com/alicevision/popsift/pull/144)
 - Added vlFeat's descriptor extraction method as an option [PR](https://github.com/alicevision/popsift/pull/167)
+- Added CI for windows on Github Actions and refactoring [PR](https://github.com/alicevision/popsift/pull/172)
 
 ### Changed
 
@@ -40,6 +41,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Remove broken and unused code path from L2 normalization [PR](https://github.com/alicevision/popsift/pull/166)
 - Do not use NVTX [PR](https://github.com/alicevision/popsift/pull/162)
 - Documentation: Biblio build [PR](https://github.com/alicevision/popsift/pull/132)
+- Fixed include for CUDA 13 [PR](https://github.com/alicevision/popsift/pull/169)
 
 ### Documentation
 

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -16,7 +16,7 @@ else()
       STRING "Which CUDA CCs to support: native, all, all-major or an explicit list delimited by semicolons")
 endif()
 
-project(PopSift VERSION 1.0.0 LANGUAGES CXX CUDA)
+project(PopSift VERSION 0.10.0 LANGUAGES CXX CUDA)
 
 # Policy to support CUDA as a first-order language for CMake.
 # Since CMake 3.18. See https://cmake.org/cmake/help/latest/policy/CMP0104.html

--- a/README.md
+++ b/README.md
@@ -1,7 +1,7 @@
 
 # PopSift
 
-[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/3728/badge)](https://bestpractices.coreinfrastructure.org/projects/3728) 
+[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/3728/badge)](https://bestpractices.coreinfrastructure.org/projects/3728)
 [![Codacy Badge](https://app.codacy.com/project/badge/Grade/64f9192b53df46b483e7cf5be7e2dddd)](https://app.codacy.com/gh/alicevision/popsift/dashboard?utm_source=gh&utm_medium=referral&utm_content=&utm_campaign=Badge_grade)
 
 PopSift is an open-source implementation of the SIFT algorithm in CUDA.
@@ -33,7 +33,7 @@ Optionally, for the provided applications:
 
 To build the library, you can run:
 
-```
+```shell
 mkdir build && cd build
 cmake ..
 make
@@ -101,7 +101,7 @@ PopSift can be configured at runtime to use constants that affect its behaviours
 In particular, users can choose to generate results very similar to VLFeat or results that are closer (but not as close) to the SIFT implementation of the OpenCV extras.
 We acknowledge that there is at least one SIFT implementation that is vastly faster, but it makes considerable sacrifices in terms of accuracy and compatibility.
 
-## Continuous integration:
+## Continuous integration
 
 * ![Continuous Integration](https://github.com/alicevision/popsift/workflows/Continuous%20Integration/badge.svg?branch=master) master branch (Linux and Windows).
 
@@ -113,28 +113,27 @@ PopSift is licensed under [MPL v2 license](COPYING.md).
 SIFT was patented in the United States from 1999-03-08 to 2020-03-28. See the [patent link](https://patents.google.com/patent/US6711293B1/en) for more information.
 PopSift license only concerns the PopSift source code and does not release users of this code from any requirements that may arise from patents.
 
-
 ## Cite Us
 
 If you use PopSift for your publication, please cite us as:
+
 ```bibtex
 @inproceedings{Griwodz2018Popsift,
-	 author = {Griwodz, Carsten and Calvet, Lilian and Halvorsen, P{\aa}l},
-	 title = {Popsift: A Faithful SIFT Implementation for Real-time Applications},
-	 booktitle = {Proceedings of the 9th {ACM} Multimedia Systems Conference},
-	 series = {MMSys '18},
-	 year = {2018},
-	 isbn = {978-1-4503-5192-8},
-	 location = {Amsterdam, Netherlands},
-	 pages = {415--420},
-	 numpages = {6},
-	 doi = {10.1145/3204949.3208136},
-	 acmid = {3208136},
-	 publisher = {ACM},
-	 address = {New York, NY, USA},
+    author = {Griwodz, Carsten and Calvet, Lilian and Halvorsen, P{\aa}l},
+    title = {Popsift: A Faithful SIFT Implementation for Real-time Applications},
+    booktitle = {Proceedings of the 9th {ACM} Multimedia Systems Conference},
+    series = {MMSys '18},
+    year = {2018},
+    isbn = {978-1-4503-5192-8},
+    location = {Amsterdam, Netherlands},
+    pages = {415--420},
+    numpages = {6},
+    doi = {10.1145/3204949.3208136},
+    acmid = {3208136},
+    publisher = {ACM},
+    address = {New York, NY, USA},
 }
 ```
-
 
 ## Acknowledgements
 

